src/main/scala/org/broadinstitute/dsde/firecloud/core/GetEntitiesWithType.scala

```diff
@@ -1,90 +0,0 @@
-package org.broadinstitute.dsde.firecloud.core
-
-import akka.actor.{Actor, Props}
-import akka.event.Logging
-import akka.pattern.pipe
-import org.broadinstitute.dsde.firecloud.core.GetEntitiesWithType.ProcessUrl
-import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
-import org.broadinstitute.dsde.rawls.model.ErrorReport
-import org.broadinstitute.dsde.firecloud.model.RequestCompleteWithErrorReport
-import org.broadinstitute.dsde.firecloud.model.ErrorReportExtensions._
-import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectiveUtils, FireCloudRequestBuilding}
-import org.broadinstitute.dsde.firecloud.service.PerRequest.{PerRequestMessage, RequestComplete}
-import org.broadinstitute.dsde.rawls.model.Entity
-import spray.client.pipelining._
-import spray.http.HttpEncodings._
-import spray.http.HttpHeaders.`Accept-Encoding`
-import spray.http.HttpResponse
-import spray.http.StatusCodes._
-import spray.httpx.SprayJsonSupport._
-import spray.httpx.encoding.Gzip
-import spray.json.DefaultJsonProtocol._
-import spray.json.JsValue
-import spray.routing.RequestContext
-
-import scala.concurrent.Future
-import scala.util.{Failure, Success}
-
-object GetEntitiesWithType {
-  case class ProcessUrl(url: String)
-  def props(requestContext: RequestContext): Props = Props(new GetEntitiesWithTypeActor(requestContext))
-}
-
-class GetEntitiesWithTypeActor(requestContext: RequestContext) extends Actor with FireCloudRequestBuilding {
```

Comment:
as noted, the code in here has been simplified and rolled into `EntityService.scala`